lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -301,19 +301,19 @@ public function get_item_schema() {
 					'context'              => array( 'view', 'edit', 'embed' ),
 					'properties'           => array(
 						'name'       => array(
-							'description' => 'Name of the font family preset, translatable.',
+							'description' => __( 'Name of the font family preset, translatable.', 'gutenberg' ),
```

Comment:
These descriptions were copied from theme.json schema and we missed adding translation functions, thanks for correcting this!